.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+- 10
+cache:
+  directories:
+  - node_modules
+script:
```

Comment:
you would need to tweak this, npm run build is available in client while npm test is available in both outer and inner module and we need to run all three